package.json

```diff
@@ -44,9 +47,12 @@
     "falcor-http-datasource": "^0.1.3",
     "falcor-router": "^0.4.0",
     "history": "^3.0.0",
+    "http": "0.0.0",
```

Comment:
it's for lib/API.js when I access the ghost API. I think it might be a built-in feature so maybe it doesn't need to be installed. Should I remove it?